airflow/dags/mental_health_dag.py

```diff
@@ -0,0 +1,129 @@
+import json
+import logging
+
+import pendulum
+import pymsteams
+import requests
+from airflow import DAG
+from airflow.hooks.base import BaseHook
+from airflow.models import Variable
+from airflow.operators.empty import EmptyOperator
+from airflow.operators.python import PythonOperator
+
+logging.basicConfig(level=logging.INFO, format="%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+logger = logging.getLogger("ContentCreationLogger")
+
+
+def load_quote(ti):
+    """Task to load a quote"""
+    api_url = Variable.get("QUOTE_API_URL")
+    try:
+        response = requests.get(api_url)
+        response.raise_for_status()
+        quote_data = response.json()
+        quote = quote_data.get("content")
+        author = quote_data.get("author")
+        if quote and author:
+            logger.info("Quote is successfully loaded")
+            ti.xcom_push(key="quote", value=f"{quote} — {author}")
+        else:
+            logger.error("Failed to load quote")
+            raise ValueError("Failed to load quote")
+    except requests.exceptions.RequestException as e:
+        logger.exception(f"Error loading quote: {e}")
```

Comment:
What is the difference between this error and previous one? Probably it would be better to combine them in one exception handling flow?